locales/en.lua

```diff
@@ -4,7 +4,7 @@ local Translations = {
         door_not_found = "Did not receive a model hash, if the door is transparent, make sure you aim at the frame of the door",
         same_entity = "Both doors can't be the same entity",
         door_registered = "This door is already registered",
-        door_identifier_exists = "A door with this identifier already exists in the config. (%s)",
+        door_identifier_exists = "A door with this identifier already exists in the config. (%{value})",
```

Comment:
This is wrong, this takes in formatting of lua, not of the locale system, please revert this

Reply:
Hey @BerkieBb, could you share a little bit more on this?

	if Config.DoorList[identifier] then
		QBCore.Functions.Notify((Lang:t("error.door_identifier_exists")):format(identifier), 'error')
		canContinue = true
		return
	if Config.DoorList[identifier] then 
		QBCore.Functions.Notify(Lang:t("error.door_identifier_exists", {value = identifier}), 'error') 
		canContinue = true 
		return 
		
The lang needs to use ${value} as it's where I am placing the "identifier", otherwise this code would not work.

Reply:
Corrected